custodian_templates/config_template.py

```diff
@@ -0,0 +1,20 @@
+from typing import Mapping
+
+
+def custodian_organizations_config_template(config: Mapping) -> Mapping:
+    dict_template = {
+        "accounts": [
+            {
+                "account_id": account["account_id"],
+                "name": account["account_name"],
+                "regions": [region for region in config["aws"]["regions"]],
+                "role": __create_role_string(account["account_id"], config["aws"]["IAM_role_name"])
+            } for account in config["aws"]["accounts"]
+        ]
+    }
+
+    return dict_template
+
+
+def __create_role_string(account_id: str, iam_role_name: str) -> str:
```

Comment:
I would strictly avoid prepending double underscores to definition names or other implicit python magic, unless you really really need that magic.  Since this seems like a utility function that's general enough to be shared, I would consider even having a `utils.py` file and importing from there (with no underscores, as a general function intended to be shared).

Reply:
It may be wise to avoid double underscores since they sometimes have meaning in other contexts. However, in this case there is no magic since functions names prefixed with double underscores are not mangled the way class method names are.

For instance, `__create_role_string` can be imported verbatim into other modules, `from custodian_templates.config_templates import __create_role_string`.

Classes mangle the names:
```
>python -c "
class Foo:
   def __doom(self):
       print('doom')
Foo()._Foo__doom()
"
doom
```

Reply:
The PEP8 for naming functions with double-underscores: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

Reply:
Removed the double underscores, kept the function in the same file for now, if I end up creating more utility functions in the future I'll move them to a separate file to keep things organized.

TIL name mangling.